.eslintrc

```diff
@@ -1,5 +1,6 @@
 {
   extends: "lob",
+  extends: "lob/migrations",
```

Comment:
You should move this line into [a new file](https://github.com/lob/lob-api/blob/master/db/migrations/.eslintrc) in the `db/migrations` directory so that these rules only apply to the files in that directory.